terraform/digital.gov.tf

```diff
@@ -563,6 +551,19 @@ resource "aws_route53_record" "demo_pra_digital_gov__acme-challenge_txt" {
   records = ["qzIXA_qU7a3io8b_FRxFVbPBUKZ83XtglufzS7qKnlg"]
 }
 
+# PRA Demo Redirect
+# redirect demo.pra.digital.gov to pra.digital.gov/
+module "demo_pra_digital_redirect_to_pra_digital" {
```

Comment:
@RileySeaburg this module is not necessary. You can remove it.

Reply:
@apburnes How do I get the "demo.pra" to redirect to "pra" do you all handle that?

Reply:
That's what the Pages redirect service handles.

Reply:
Removed